frontend/src/components/cfp-form/index.tsx

```diff
@@ -380,13 +382,15 @@ export const CfpForm: React.SFC<Props> = ({
           description={<FormattedMessage id="cfp.abstractDescription" />}
           errors={getErrors("validationAbstract")}
         >
-          <Textarea
+          <LimitedTextarea
             sx={{
               resize: "vertical",
               minHeight: 200,
             }}
             {...textarea("abstract")}
             rows={6}
+            maxLength={5000}
+            limit={5000}
```

Comment:
we already pass `limit` to `LimitedTextarea`, why do we have to specify it twice? Why doesn't `LimitedTextarea` do it? In what case do we want to show a limit and enforce another?

```suggestion
            limit={5000}
```

I would prefer if the LimitedTextarea was responsible for enforcing it so we are consistent

Reply:
Yeah agree to pass only one of them, I will keep `maxLength` only as it's already used for TextArea

Reply:
👍  let's replace limit with maxLength then